ehsm_kms_service/payload_checker.js

```diff
@@ -49,7 +49,7 @@ const keyid = {
 }
 const aad = {
   type: PARAM_DATA_TYPE.BASE64,
-  maxLength: MAX_LENGTH,
+  maxLength: MAX_LENGTH * 4 / 3,  // This is the length after base64 encoding.

```

Comment:
any relation to the fix? if not, please create a new patch for it.

Reply:
already clear this